src/app/layouts/button/button.component.ts

```diff
@@ -8,6 +8,7 @@ import { Component, Input, Output, EventEmitter } from '@angular/core';
 export class ButtonComponent {
     @Input() type: string;
     @Input() disabled = false;
+    @Input() hidden = false;
```

Comment:
It seems that we do not need this property. As I see it used to hide the button on the export-popup.component.html. The better solution is to hide app-button component on the export-popup.component.html directly (`*ngIf="selectedIndex"` or smth like this, for example).

Reply:
I wouldn't change that right now, maybe we will change the design of this block and use a checkbox instead of a button.